soda/scientific/tests/distribution_comparison_test.py

```diff
@@ -120,53 +104,6 @@ def test_distribution_checker(method, reference_file_path, test_data, expected_s
     assert check_results["check_value"] == pytest.approx(expected_p, abs=1e-3)
 
 
-@pytest.mark.parametrize(
```

Comment:
As mentioned in DMs. We should:
- use the execute with allowing for errors in a separate test that tests for the errors specifically
- and then peek into the cloud result logs for a specific log message using `pop_scan_result()